INSTALLER-HACKING.md

```diff
@@ -0,0 +1,63 @@
+# Running local builds with the installer
+
+If you want to run a local build with the installer, you need to do a bit of hacking. There is a script that will do most of the dirty work for you.
+
+## Step 0: Prepare your credentials
+
+You will need to do this once, and every time your token resets.
```

Comment:
This sentence is very mysterious on the first read-through...

And are you sure this is needed? I just tried `podman pull registry.svc.ci.openshift.org/openshift/origin-release:v4.0` and it worked without me logging in.

Anyway, if it is sometimes necessary, I was going to suggest that this could just be moved into the script; if the `podman pull` fails, ask the user to go to the CI server console, copy the login command, and paste it into the terminal. Then the script can extract the token from that and run `podman login` and then try again.

Reply:
Looks like it was only needed for me, because I logged in to push to my repository, and the token expired. Neat.